com.github.scrivano.scrivano.yml

```diff
@@ -0,0 +1,31 @@
+app-id: com.github.scrivano.scrivano
```

Comment:
the app-id doesn't meet the requirement. There is no `scrivano` user or organization on Github.

Reply:
Ok I changed the id to meet the requirements